### DISCUSSION TITLE
Cannot create list in 'Stars' lists

### DISCUSSION THREAD

**Discussion by youngerous** [community/community#8605] [Other Feature Feedback, Questions, & Ideas] [answered]
When I try to create a new list by clicking a '+ Create list' button beside each repository, its color becomes gray but nothing happens :(

By the way, thanks for providing useful function!

![2021-12-10_02-07-23](https://user-images.githubusercontent.com/38049006/145443141-84989ad5-3bcc-4bb4-98a1-29db55815eb4.png)

**Comment by bailey** [comment, marked as answer]
Hi @youngerous! Thanks so much for logging this. We've logged the bug and are working on a fix as we speak. 

As we are seeing this intermittently, refreshing and trying again might work for you in the meantime. We'll let you know once a fix is out. ✨

**Reply by toddsiegel** [reply]
First, let me say that I have wanted this for a long time, so thank you for releasing it!

That said, it is pretty consistently broken for me too on the repo page: in Brave (Version 1.32.113 Chromium: 96.0.4664.45 (Official Build) (x86_64)) and FF 95. Refreshing does not seem to help either. 🙁 

On the `/stars` page is seems to sometimes work. 😕

**Comment by pavanpej** [comment]
I can confirm that this behavior is consistent for me over the last 2 hours when I've been trying intermittently. 
Very useful feature though. 👍🏼
